nservicebus/upgrades/support-policy.md

```diff
@@ -91,4 +91,4 @@ Component packages, such as [transports](/transports/), [persisters](/persistenc
 
 The support dates for NServiceBus and each component package can be found on the [supported versions](supported-versions.md) page.
 
-<script type="text/javascript" src="supported-versions.js"></script>
\ No newline at end of file
+<script type="text/javascript" src="supported-versions.js"></script>
```

Comment:
I did not add this. I have no idea where it came from. I updated the table in the browser :o